flink-connectors/flink-connector-hbase-2.2/src/test/java/org/apache/flink/connector/hbase2/HBaseConnectorITCase.java

```diff
@@ -250,13 +252,25 @@ public void testTableSink() throws Exception {
                         + " FROM "
                         + TEST_TABLE_1;
 
+        TableResult tableResult = tEnv.executeSql(query);
+
         // wait to finish
-        tEnv.executeSql(query).await();
+        tableResult.await();
+
+        assertEquals(
+                "Expected INSERT rowKind", RowKind.INSERT, tableResult.collect().next().getKind());
 
         // start a batch scan job to verify contents in HBase table
         TableEnvironment batchEnv = TableEnvironment.create(batchSettings);
         batchEnv.executeSql(table2DDL);
 
+        Table countTable =
+                batchEnv.sqlQuery(
+                        "SELECT " + "  count(h.rowkey) " + "FROM " + TEST_TABLE_2 + " AS h");
+
+        // check the count is right
```

Comment:
I think the comment is superfluous, the assertion is obvious.

Reply:
sure, thanks